easybuild/easyblocks/c/clang.py

```diff
@@ -407,21 +408,58 @@ def build_with_prev_stage(self, prev_obj, next_obj):
         mkdir(next_obj)
         change_dir(next_obj)
 
-        # Configure.
-        CC = os.path.join(prev_obj, 'bin', 'clang')
-        CXX = os.path.join(prev_obj, 'bin', 'clang++')
+        # Make sure clang and clang++ compilers from the previous stage are (temporarily) in PATH
```

Comment:
@casparvl Maybe we should clarify why this is required (so `prepare_rpath_wrappers` can find the compiler commands)?

Reply:
Well, yes and no. By now, it's _also_ to make sure that `which` will be able to find them (I took out your `which ... or os.path.join(...)` construct, since that was now duplicate. So, in the current code, there's two reasons...
But, I will simply name both reasons here, that way, we know why we do it like this.

Reply:
@casparvl Test reports check out, so as soon as the additional comments are there, this is good to go!

Reply:
Done!